<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer_Options_Dialog</class>
 <widget class="QDialog" name="Viewer_Options_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer Options</string>
  </property>
  <widget class="QPushButton" name="RESET_pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset to defaults</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CANCEL_pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Default Editor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="defaultEditor_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Used Editor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="userdefinedEditor_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_uderdefinedEditor_pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_defaultEditor_pushButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
